Homework7/Book.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Homework7
+{
+    interface IReadable 
+    {
+        string? Description { get; }
+        void TurnPage();
+    }
+    interface IRecyclable
+    {
+        void Recycle();
+    }
+    internal class Book:IReadable, IRecyclable
```

Comment:
Leave empty lines between: classes, constructors, properties/fields and constructors, methods.
No need empty lines between fields or properties

Reply:
fixed (hope so)